src/components/Person/Person.jsx

```diff
@@ -1 +1,44 @@
-// export const Person = ({ person }) => ();
+// export const Person = ({ person }) => ();import
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const role = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${role}`
+          : `I am not married`}
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  age: null,
+  partnerName: null,
+};
+
+Person.propTypes = {
+  name: PropTypes.string.isRequired,
+  age: PropTypes.number,
+  sex: PropTypes.oneOf(['m', 'f']).isRequired,
+  isMarried: PropTypes.bool.isRequired,
+  partnerName: PropTypes.string,
+};
```

Comment:
you don't get `name` and `age`, ... as props in your component, you only have person, this propsTypes are not correct